examples/all-clusters-app/esp32/README.md

```diff
@@ -97,7 +97,7 @@ make sure the IDF_PATH has been exported(See the manual setup steps above).
         $ cd ${HOME}/tools/esp-idf
         $ ./install.sh
         $ . ./export.sh
-        $ cd {path-to-connectedhomeip}
+        $ cd {path-to-connectedhomeip}/examples/all-clusters-app/esp32/
```

Comment:
I'm not sure, but if I would follow this guideline, after `cd`ing into the esp32 directory later commands will not work, because they assume that we are in the Matter home directory. E.g. `source ./scripts/bootstrap.sh` in the "download and install packages" section.

Reply:
Weird, for me when I was in the matter root directory I kept getting an error about the build directory not being deletable, it was only after I cd'ed that it worked; some other folks here at Google (Kevin, Daniel) experienced the same

I'm on mac (& I believe they are too), could that be the issue?

I'm also synced to latest & only started seeing the issue after syncing

Reply:
1. In `{path-to-connectedhomeip}` folder, issue `source ./scripts/bootstrap.sh` to setup CHIP build env;
2. In example folders, like `{path-to-connectedhomeip}/examples/all-clusters-app/esp32/`, issue `idf.py set-target esp32` and following cmds to build ESP examples.

To reslove your issue, please add `cd {path-to-connectedhomeip}/examples/all-clusters-app/esp32/` before `idf.py set-target esp32`. Could you help to add it to all ESP examples readme?